src/test/resources/data/functional/controllers/accountTaskController/updateAccountTask.scenarios.json

```diff
@@ -0,0 +1,65 @@
+[
+    {
+      "description": "Should successfully update the task for given task id", 
+      "input": {
+        "body": {
+          "crm_organization_user_id":"0011e00000bWSxdAA1",
+          "description":"Update remaining documentations",
+          "due_date":"2023-12-01"
+        },
+        "accountId": "0011e00000bWSxdAAG",
+        "taskId":"0691e000001X1yTAAS"
```

Comment:
Add Test Cases for description/due date/crm_organization_user_id

Reply:
Added